.travis.yml

```diff
@@ -1,44 +0,0 @@
-language: php
```

Comment:
It's going to be changed to a javascript environment (at least for now), in #88. So I propose to just remove it here